routes/api/volunteer.js

```diff
@@ -24,6 +25,38 @@ module.exports = {
     activate: function (apiRouter) {
         const volunteerRouter = express.Router();
 
+        /**
+         * @api {get} /volunteer/:id get a volunteer's information
+         * @apiName getVolunteer
+         * @apiGroup Volunteer
+         * @apiVersion 1.3.0
+         * 
+         * @apiParam (param) {ObjectId} id a volunteer's unique mongoID
+         * 
+         * @apiSuccess {String} message Success message
+         * @apiSuccess {Object} data Volunteer object
+         * @apiSuccessExample {object} Success-Response: 
+         *      {
+                    "message": "Successfully retrieved volunteer information", 
+                    "data": {...}
+                }
+
+         * @apiError {String} message Error message
+         * @apiError {Object} data empty
+         * @apiErrorExample {object} Error-Response: 
+         *      {"message": "Volunteer not found", "data": {}}
+         */
+        volunteerRouter.route("/:id").get(
+            Middleware.Auth.ensureAuthenticated(),
+            Middleware.Auth.ensureAuthorized([Services.Volunteer.findById]),
+
+            Middleware.Validator.RouteParam.idValidator,
```

Comment:
The `RouteParam.idValidator` should be before `ensureAuthorized` so that there is not a 500 error when the user enters in an invalid id.